client/index.html

```diff
@@ -46,7 +46,7 @@
     <script type="text/javascript">
       window.CELLXGENE = {};
       window.CELLXGENE.API = {
-        prefix: `http://localhost:<%= CXG_SERVER_PORT %>${location.pathname}/api/`,
```

Comment:
This change was needed to get explorer to work locally. Without this change, the explorer client will make API requests out to `http://localhost:5005/d/test.cxg//api/v0.3/config`, and the `//` in the URL would result in a 404. See [slack thread](https://czi-sci.slack.com/archives/C023Q1APASK/p1698700604868129) for more context.